spec/models/registration_spec.rb

```diff
@@ -509,10 +508,22 @@
         expect(reg.waiting_list_position).to eq(6)
       end
 
+      it 'no change if we try to add a registration on the waiting list' do
+        reg1.update_lanes!({ user_id: reg1.user.id, competing: { status: 'waiting_list' } }.with_indifferent_access, reg1.user)
+
+        expect(reg1.waiting_list_position).to eq(1)
+        expect(reg2.waiting_list_position).to eq(2)
+        expect(reg3.waiting_list_position).to eq(3)
+        expect(reg4.waiting_list_position).to eq(4)
+        expect(reg5.waiting_list_position).to eq(5)
+
+        expect(waiting_list.entries.count).to eq(5)
+      end
+
       it 'removes from waiting list' do
         reg4.update_lanes!({ user_id: reg4.user.id, competing: { status: 'pending' } }.with_indifferent_access, reg4.user)
 
-        expect(reg4.waiting_list_position).to eq(nil)
+        expect(reg4.reload.waiting_list_position).to eq(nil)
```

Comment:
Curious: Why is a `reload` necessary here when it wasn't necessary before?

Reply:
Apparently is isn't? Fixed

Reply:
Speculation: LIkely added the reload in an attempt to fix the test which was failing for an unrelated reason, and forgot to remove it once the test was passing again